include/plugin/runtime-support.h

```diff
@@ -278,14 +310,28 @@ void reject_protocol(const std::string& reason);
  */
 void protocol_begin(const std::optional<std::string>& analyzer);
 
+/**
+ * Gets a handle to the given child analyzer. If the child does not yet exist
+ * it will be created.
+ *
+ * @param analyzer the Zeek-side name of the analyzer to get
```

Comment:
```suggestion
 * Gets a handle to a child analyzer of a given type. If a child of that type does not yet exist
 * it will be created.
 *
 * @param analyzer the Zeek-side name of the analyzer to get (e.g., `HTTP`)
```